modules/extension/xsd/xsd-core/src/main/java/org/geotools/xml/impl/XsLocalDateFormat.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2004-2016  The Apache Software Foundation
```

Comment:
Question about this one ... it is a new file does it need a new header?

Okay I see what happened, will fix during merge.

```
/*
 * (C) 2011-2016 Open Source Geospatial Foundation (OSGeo)
 * (C) 2004  The Apache Software Foundation
 * 
 ...
```